### ISSUE TITLE
Clarify whether disclosure's toggle button should use aria-pressed

### ISSUE THREAD

**Issue by 0ddfell0w** [w3c/aria-practices#897]
[The APG entry for Disclosure](https://www.w3.org/TR/wai-aria-practices-1.1/#disclosure) mentions that aria-expanded should be set on the toggle button. Should the toggle button also use aria-pressed, as is recommended in the [entry for button](https://www.w3.org/TR/wai-aria-practices-1.1/#button)? 

In either case, could it be worth clarifying that in the APG? Thanks!

**Comment by Decrepidos** [comment]
Personally I do not believe aria-pressed is needed for a disclosure show/hide; as aria-expanded is sufficient to indicate the expanded/collapsed state for assistive technology users. I find when testing sites that implement both it sometimes gets confusing because of the how browser/ATs’ announce the two attributes when used together. Additionally when viewing JAWS form fields dialog the state of pressed is displayed for the disclosure button which IMO is not necessary and may seem verbose. 😁
